gguf-py/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "gguf"
-version = "0.9.0"
+version = "0.9.1"
```

Comment:
Normally we don't bump version when new arch is added, so I'm not sure if this change is necessary